### PR TITLE
docs(env): update .env.example

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Refresh `.env.example`. Disable exchange per default and add one missing env.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,8 +1,15 @@
 VITE_BITCOIN_MAINNET=false
 VITE_ETHEREUM_MAINNET=false
+VITE_ETHERSCAN_API_URL=https://api-sepolia.etherscan.io/api
```
Comment (AntonioVentilii-DFINITY) [inline .env.example:3]: I cannot find a usage for this variable

File: .env.example [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
 VITE_ETHERSCAN_API_KEY=
+VITE_ETHERSCAN_NETWORK=sepolia
```
Comment (AntonioVentilii-DFINITY) [inline .env.example:5]: I cannot find a usage for this variable

File: .env.example [hunk 3]
```diff
@@ -1,8 +1,15 @@
...
 VITE_INFURA_API_KEY=
+VITE_INFURA_NETWORK=sepolia
```
Comment (AntonioVentilii-DFINITY) [inline .env.example:7]: I cannot find a usage for this variable

File: .env.example [hunk 4]
```diff
@@ -1,8 +1,15 @@
...
 VITE_ALCHEMY_API_KEY=
+VITE_ALCHEMY_NETWORK=eth-sepolia
+VITE_ALCHEMY_JSON_RPC_URL=https://eth-sepolia.g.alchemy.com/v2
+VITE_ETHEREUM_CHAIN_ID=11155111
+VITE_ETHEREUM_SYMBOL=SepoliaETH
```
Comment (AntonioVentilii-DFINITY) [inline .env.example:12]: I cannot find a usage for the 4 new variables above

File: .env.example [hunk 5]
```diff
@@ -14,6 +21,13 @@ VITE_OISY_ALPHA_WARNING_URL="https://github.com/dfinity/oisy-wallet#status"
...
 VITE_COINGECKO_API_URL=https://api.coingecko.com/api/v3
-VITE_EXCHANGE_DISABLED=false
+VITE_COINGECKO_API_KEY=
+VITE_EXCHANGE_DISABLED=true
+VITE_ETHEREUM_EXPLORER_URL=https://sepolia.etherscan.io
+VITE_BITCOIN_EXPLORER_URL=https://blockstream.info/testnet
+VITE_ICP_EXPLORER_URL=https://dashboard.internetcomputer.org
+VITE_CKBTC_EXPLORER_URL=https://dashboard.internetcomputer.org/bitcoin
+VITE_CKETH_EXPLORER_URL=https://dashboard.internetcomputer.org/ethereum
+VITE_BTC_NETWORK=testnet
```
Comment (AntonioVentilii-DFINITY) [inline .env.example:31]: I cannot find a usage for the new variables above except for `VITE_COINGECKO_API_KEY` and `VITE_EXCHANGE_DISABLED`

**Review by AntonioVentilii-DFINITY — COMMENTED** [review]
I cannot find the usage of some of them

File: .env.example (hunk 5 above)
Comment (peterpeterparker) [reply .env.example]: Thanks @AntonioVentilii-DFINITY . Those were left over from the original way to configure the env which were remaining on my side. I cleaned up my env and here adding the one missing.

**Review by AntonioVentilii-DFINITY — APPROVED** [review]

**Pull request merged**